x/liquidstakeibc/migrations/v4/store.go

```diff
@@ -0,0 +1,61 @@
+package v4
```

Comment:
uh, can we not just write these parts as upgrade instructions? and not increase module consensus version? 
Module consensus version is usually changed when store structs change. not when we edit/ fork entries